frontend/wallet/src/render/views/settings/DelegationSettings.re

```diff
@@ -113,32 +113,36 @@ let make = (~publicKey) => {
            {React.string("Delegate Participation To")}
          </div>
          <Spacer height=1. />
-         <TextField
-           label="Key"
-           value={
-             Option.map(~f=PublicKey.toString, state.delegate)
-             |> Option.withDefault(~default="")
-           }
-           mono=true
-           onChange={value =>
-             changeState(_ =>
-               {
-                 delegate: Some(PublicKey.ofStringExn(value)),
-                 fee: state.fee,
-               }
-             )
-           }
-         />
+         <div className={Css.style([Css.maxWidth(`rem(40.))])}>
```

Comment:
Should we lift this style to a variable in `Styles` above so we get rid of the code duplication? Also, we probably want to always keep these two style calls in sync so it seems like a good idea

Reply:
👍

Reply:
lifted